ghproxy/README.md

```diff
@@ -3,4 +3,18 @@
 ghProxy is a reverse proxy HTTP cache optimized for use with the GitHub API (https://api.github.com).
 It is essentially just a reverse proxy wrapper around [ghCache](/ghproxy/ghcache) with some additional prometheus instrumentation logic to monitor disk usage and push metrics to a prometheus push gateway.
 
-ghProxy is designed to reduce API token usage by allowing many components to share a single ghCache. Note that components must use the same API token to benefit from the cache and avoid clobbering existing cache entries for other tokens.
\ No newline at end of file
+ghProxy is designed to reduce API token usage by allowing many components to share a single ghCache. Note that components must use the same API token to benefit from the cache and avoid clobbering existing cache entries for other tokens.
+
+## How to use
+
+To use ghProxy for your prow instance,
+
+- Create a `StorageClass` using [`gce-ssd-retain_storageclass.yaml`](/prow/cluster/gce-ssd-retain_storageclass.yaml)
+if on GCE or specify your own storage class.
+- Create a `PersistentVolumeClaim` and `Deployment` using the config [here](prow/cluster/ghproxy_deployment.yaml).
+- Finally, create a [`Service`](/prow/cluster/ghproxy_service.yaml).
+- In the deployments for your prow components, use `--github-endpoint=http://ghproxy` as shown
```

Comment:
We should explicitly mention that this should be done for all prow components that use the main Prow bot token. (i.e. deployments that use the `oauth-token` secret).